src/js/index.js

```diff
@@ -53,6 +53,9 @@ const setupHeader = () => {
             if (navLabel) {
                 navLabel.classList.toggle('is-open');
             }
+
+            document.documentElement.classList.toggle('is-navInView');
```

Comment:
Looks like this class is permanently added once the menu button is clicked, is that right?

Reply:
Also, would be good to assign `document.documentElement` to a variable called something like `html` as I had to go and look up what it would actually select ✏️

Reply:
@tommaton ^

Reply:
I missed the `toggle` function call 🤦‍♂️